firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLitePersistence.java

```diff
@@ -326,13 +347,13 @@ private void ensureConfigured(SQLiteDatabase db) {
     @Override
     public void onCreate(SQLiteDatabase db) {
       ensureConfigured(db);
-      new SQLiteSchema(db, serializer).runMigrations(0);
+      dataMigrations = new SQLiteSchema(db, serializer).runMigrations(0);
```

Comment:
I think we need some new names here. We are using migrations in two different context. Should we rename the existing migration functionality to `runSchemaUpgrades()`?.

It is also pretty surprising that this returns such an important return type. Did you consider something like this:

```
SQLiteSchema db = new SQLiteSchema(db, serializer);
db.runSchemaUpgrade(0);
dataMigrations = db.getDataMigrations(0);
```


```